muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/application-sub-panel.tsx

```diff
@@ -31,26 +30,74 @@ const ApplicationSubPanel: React.FC<SubPanelProps> = (props) => {
         modifier ? `application-sub-panel--${modifier}` : ""
       }`}
     >
-      <div
-        className={`application-sub-panel__header ${
-          modifier ? `application-sub-panel__header--${modifier}` : ""
-        }`}
-      >
-        {title}
-      </div>
-      <div
-        className={`application-sub-panel__body ${
-          modifier ? `application-sub-panel__body--${modifier}` : ""
-        } ${
-          bodyModifier ? `application-sub-panel__body--${bodyModifier}` : ""
-        }`}
-      >
-        {children}
-      </div>
+      {children}
     </div>
   );
 };
 
+/**
+ * ApplicationSubPanelHeader
+ * @param props component props
+ * @returns JSX.Element
+ */
+const ApplicationSubPanelHeader: React.FC<{ modifier?: string }> = (props) => (
+  <div
+    className={`application-sub-panel__header ${
+      props.modifier ? `application-sub-panel__header--${props.modifier}` : ""
+    }`}
+  >
+    {props.children}
+  </div>
+);
+
+/**
+ * SubPanelViewHeaderProps
+ */
+interface SubPanelViewHeaderProps {
+  title: string;
+  titleDetail?: string;
+  modifier?: string;
+  aside?: string | JSX.Element;
+}
+
+/**
+ * ApplicationSubPanelViewHeader
+ * @param props component props
+ * @returns JSX.Element
+ */
+export const ApplicationSubPanelViewHeader: React.FC<
+  SubPanelViewHeaderProps
+> = (props) => (
+  <>
+    <div className="application-sub-panel__header-main-container">
+      <h2 className="application-sub-panel__header-main application-sub-panel__header-main--guider-profile-student-name">
```

Comment:
Same thing here. Modifier is --guider-profile-student-name though component itself is not spefically pointed to be component for only that use